docs/changelog.txt

```diff
@@ -1,7 +1,41 @@
 [[changelog]]
 [appendix]
-== Release Notes
+Release Notes
```

Comment:
This is undoing the change that I made to use the recommended style of header. The correct version here is the current version on `master`:

> == Release Notes

Similarly for the other header below.

Should I submit a change to the `0.1` branch to fix this and we can then merge with that fix, or do you want to merge this PR, and I'll submit separate fixes to `master` and `0.1` branches?

LGTM outside of these two concerns.